windows/micropython.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
That's not the case, and other IDE's present the same issue (Android Studio comes to mind). Newer version of the IDE tries to convert the project to the latest version, if possible in a backwards-compatible way. Depending on the project type and version diff, converting the project might fail or it may end up being incompatible with older versions of the IDE.